service/matching/physical_task_queue_manager.go

```diff
@@ -387,19 +503,14 @@ func (c *physicalTaskQueueManagerImpl) LegacyDescribeTaskQueue(includeTaskQueueS
 	return response
 }
 
-func (c *physicalTaskQueueManagerImpl) Describe() *taskqueuespb.PhysicalTaskQueueInfo {
-	return &taskqueuespb.PhysicalTaskQueueInfo{
-		Pollers:        c.GetAllPollerInfo(),
-		TaskQueueStats: c.GetStats(),
-	}
-}
-
 func (c *physicalTaskQueueManagerImpl) GetStats() *taskqueuepb.TaskQueueStats {
 	return &taskqueuepb.TaskQueueStats{
 		ApproximateBacklogCount: c.backlogMgr.db.getApproximateBacklogCount(),
-		ApproximateBacklogAge:   nil,        // TODO: Shivam - add this feature
-		TasksAddRate:            float32(0), // TODO: Shivam - add this feature
-		TasksDispatchRate:       float32(0), // TODO: Shivam - add this feature
+		ApproximateBacklogAge:   durationpb.New(c.backlogMgr.taskReader.getBacklogHeadAge()), // using this and not matcher's
```

Comment:
@dnr there is a comment here for why we use reader's backlog age and not matcher's. Also, a comment is added to matcher's getBacklogAge. Reader's GetBacklogAge isn't doing anything special IMO.
If you think more clarification is needed would you please specify what comment and where it should be added?